#!/usr/bin/env python
PACKAGE = "ueye2"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("red_gain", double_t, 0, "Red channel gain", 1, 1, 6)
gen.add("green_gain", double_t, 0, "Green channel gain", 1, 1, 6)
gen.add("blue_gain", double_t, 0, "Blue channel gain", 1, 1, 6)
gen.add("auto_reference", double_t, .5, "Auto exposure/gain setpoint", .5, 0, 1)

exit(gen.generate(PACKAGE, "ueye2", "UEye"))
